GPSTest/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
 
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 21
+        targetSdkVersion 27
```

Comment:
This should still crash on initial install and run.

Reply:
🤔 guess ill change it back then

the only way to know for sure is to try it; it ran fine on my phone so

Reply:
This is related to run-time vs. install-time permissions.  If you already have the current version of the app installed and then make this change and install the new version it will work fine, because you've already accepted the permissions at install-time.  If, however, you uninstall the app, and then install it with this change, it will crash because it hasn't been configured for run-time permissions yet.

Reply:
last i recall, i installed it as a new app (because the apks i posted were signed with a different key), and it ran just fine, unless thats different from what you're referring to